README.md

```diff
@@ -42,6 +42,25 @@ hooks('ts').push(function({content}){
 /* transpile your script code here */
 });
 ```
+
+For custom blocks in `.vue` files, add other require hooks for the name of the blocks,
+prefixed by `vue-block-`. For example, to load a custom `<json></json>` block:
+
+```javascript
+const hooks = require('require-extension-hooks');
+hooks('vue').plugin('vue');
+hooks('vue-block-json').push(({ content }) => {
+    // Strings returned here will be appended to the compiled .vue file. This should return JavaScript, in a string.
+    return '';
```

Comment:
Can you have this example show something more realistic than an empty string?

Something like `export const jsonBlock = ${content}`

Another option which might be good would be to export this string as a constant from the package for people to use. So your hook would look something like `${COMPONENT_EXPORT}.name = ${content}`